packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -129,8 +134,9 @@ export const getStyles = (props: IToggleStyleProps): IToggleStyles => {
                 },
               ],
               [HighContrastSelector]: {
-                backgroundColor: 'WindowText',
+                backgroundColor: 'Highlight',
```

Comment:
could you make the similar fix to `toggle.scss` in `react-next`?
we should add the styles in `.pill &.checked` section and `@include high-contrast-edge-chromium-no-adjust;` is how you achieve the same as `getEdgeChromiumNoHighContrastAdjustSelector`